src/commands/plugins/install.ts

```diff
@@ -57,6 +57,8 @@ e.g. If you have a core plugin that has a 'hello' command, installing a user-ins
       })
       try {
         if (p.type === 'npm') {
```

Comment:
Good approach! But you are only checking for plugins from npm, if you install from a url (`else` statements) you will successfully install the plugin, here is an example:

![image](https://user-images.githubusercontent.com/20549906/110496191-03329200-80d4-11eb-9c6b-8ef55a9d73ff.png)